frontend/utils/nativeNotification.js

```diff
@@ -5,7 +5,7 @@ function checkNotification (): boolean {
 export async function requestNotificationPermission (force: boolean = false): Promise<null | string> {
   if (!checkNotification() ||
     (!force && Notification.permission !== 'default') ||
-    (force && Notification.permission !== 'granted')) {
+    (force && Notification.permission === 'granted')) {
```

Comment:
I was unsure about this too, but assumed I likely just didn't understand correctly